### DISCUSSION TITLE
Integrating data visualization

### DISCUSSION THREAD

**Discussion by henriquecbuss** [cambiatus/frontend#807] [Ideas]
On 09/08, we had a full team session with @heltonlr, where we discussed all of the data visualization we're doing (Postico, Metabase, Knime) on Cambiatus.

These tools can help us and our communities to make more informed decisions, prioritize on certain things, and just know how the platform is working and what our users are doing within it.

Folks from the P4C community already asked for some of this to be integrated into our app (https://github.com/cambiatus/frontend/issues/749), and I think we can do some really interesting things with this.

We can decide to integrate this into our app (we could use [elm-charts](https://elm-charts.org), which produces beautiful graphs), or we could have a separate platform (like Metabase) to show all of this data. Or maybe even both of them!

**Comment by lucca65** [comment]
Amazing! Hope we keep this integration tight!

**Reply by muguika** [reply]
Important to note here that the data visualization should have two fronts, at least in my opinion:

1. What @NeoVier mentioned which is focused in our members, communities, users in general. The one we have already on Metabase or integrated into the app.

2. A second one focused on external people communities that are thinking on using Cambiatus, partners, potential funders, etc. Right now we have [this page](https://www.cambiatus.com/our-numbers) displayed publicly and updated monthly, and that has done the trick partially so far, but I think a more dynamic public dashboard would be fantastic. Something like [this one](https://dashboard.sarafu.network) from Grassroots Economics. Take a look at their [network](https://viz.sarafu.network) visualization too.  

What do you think @heltonlr

**Reply by heltonlr** [reply]
Totally agree, @muguika . We can focus on the potential funders and communities that are meeting Cambiatus. Great, we already have a persona to work and prioritize. Let’s see if everyone are according next week.

Good examples too, we can do better 💪

**Reply by muguika** [reply]
Another potential audience are researchers that could be interested in studying Cambiatus

**Comment by MatheusBuss** [comment]
Perhaps to aid us in future decisions we could get some data for our own consumption. We could look into things like which devices, browsers and OS's our users access Cambiatus with. Of course this is a path to be tread lightly since we should always treat this type of data with respect and responsibility.

---

Also with the new release deployed we have a potential to track something really interesting. Product categories could really help communities to better decide how to manage their stores. Having these "tags" on products could really facilitate tracking what kind of product a community really is about, especially if we have some sort of data visualization around it.

**Reply by henriquecbuss** [reply]
We used to get device, browser, OS, etc. from Plausible, but I think we don't anymore. We have #813 to try out Matomo, which should get us that kind of data. This data is usually anonymized, so it's ok to have it (we can't link anything back to a specific person)

**Reply by lucca65** [reply]
We also get some data from Sentry, which gives us device information. Although the focus is errors, we can still get useful information about our users, and what's affecting them the most
